src/main/java/com/powsybl/openloadflow/ac/PhaseControlOuterLoop.java

```diff
@@ -33,47 +33,72 @@ public String getType() {
 
     @Override
     public OuterLoopStatus check(OuterLoopContext context) {
```

Comment:
In this outerloop we do more than just check in this method: we also switch off/on phase control. Shouldn't it be done in another method, or is it just a naming issue?